pkgs/top-level/all-packages.nix

```diff
@@ -48,7 +48,22 @@ in
       lib.mapNullable (rs: rs ++ [ bintools ]) (stdenv.allowedRequisites or null);
   };
 
-  stdenvNoLibs = mkStdenvNoLibs stdenv;
+  stdenvNoLibs =
+    if stdenv.hostPlatform != stdenv.buildPlatform && (stdenv.hostPlatform.isDarwin || stdenv.hostPlatform.isDarwin.useLLVM or false)
```

Comment:
`stdenv.hostPlatform.isDarwin.useLLVM or false``stdenv.hostPlatform.isDarwin.useLLVM or false`

should be

`stdenv.hostPlatform.isDarwin.useLLVM or false``stdenv.hostPlatform.useLLVM or false`

```
nix-repl> pkgsCross.x86_64-freebsd.stdenv.hostPlatform.useLLVM
true
```

Reply:
Oops! Thanks for catching!

Reply:
https://github.com/NixOS/nixpkgs/pull/274047